pages/learn/index.vue

```diff
@@ -52,14 +52,14 @@ export default class LearnPage extends QiskitPage {
       }
     },
     {
-      title: SOCIAL_MEDIA.stack.label,
-      description: `Have a question? Ask it on stack exchange! Qiskit
-      advocates and core contributors monitor this forum and will happily
-      answer your questions.`,
+      title: SOCIAL_MEDIA.support.label,
+      description: `If you need support on anything relating to using and
+      developing Qiskit, you can use the following forums to direct your
+      issue / queries to the appropriate audience.`,
```

Comment:
I don't understand why the description string here needs to be wrapped in back ticks ``` ` ``` rather than single quotes `'`. When I used single quotes, the linter complained https://github.com/Qiskit/qiskit.org/pull/3067/commits/e870ab8c335b8201895eeabb6c5dbf21a4db34b8

Reply:
Looks like back ticks are required if the string is written hard wrapped to multiple lines. I changed to use single quote and no wrapping in https://github.com/Qiskit/qiskit.org/pull/3067/commits/6d1733482986683d038949bdc860a8afdb8e7485